src/drone/proxy/certs.rs

```diff
@@ -75,7 +75,9 @@ impl CertRefresher {
         };
 
         for path in key_cert_path_pair.parent_paths() {
-            watcher.watch(&path, RecursiveMode::NonRecursive)?;
+            watcher
+                .watch(&path, RecursiveMode::NonRecursive)
+                .context("Error installing watcher.")?;
```

Comment:
Where exactly do these `context` strings go when the `?` "fails"?

Reply:
When an error is reported (technically, when its debug formatter is called), the context message is added to the output along with the original error.